vehicle:
  gear_ratio: 5.62
  wheel_rad: 0.0323
  max_steering_angle: 0.523
  min_turning_radius: 0.5
  mass: 2.35
  I_zz: 0.045

  wheelbase: 0.257
  wheelbase_f: 0.067
  wheelbase_r: 0.190
  trackwidth: 0.165

vesc_interface:
  speed_gain: -1660.0
  speed_offset: 0.0

  steering_gain: 0.40972
  steering_offset: 0.468

  tix_gain: -166.0

  #max_motor_accel: 2.5 # meters/second^2
  #motor_smoother_rate: 75.0 # messages/sec
  
  max_servo_speed: 3.2 # radians/second
  #servo_smoother_rate: 75.0 # messages/sec
